agent/agent/config.py

```diff
@@ -0,0 +1,173 @@
+"""Submission core configuration parameters."""
```

Comment:
With these config files, it would be nice to include a pointer to our deployment docs.